### ISSUE TITLE
NPC Behaviour infight [Parry-Haste Problem on Bosses]

### ISSUE THREAD

**Issue by Lyn3452** [SunwellWoW/Sunwell-TBC-Bugtracker#4516]
Description: I realized that NPC's especially in Hyjal have weird movement and **ALWAYS** want to stay in front of the Enemy/Boss. This makes Boss encounters (especially in Horde Camp Hyjal) way harder, because Parry-Haste gets triggered way more often and Tanks getting hard destroyed by that. (Tanking Kaz'rogal and Thrall and his 2 Shamans attacking him in front and triggering Parry-Haste while Tank is stunned is often almost a Oneshot). Even when you turn the Boss NPC's will follow and position again in front of the Boss. Would be a great fix to solve that.

How it works: NPCs are trying to stand in front of the Enemy / Boss all the time.

How it should work: NPC's shouldnt focus on standing in front of their Enemy or Boss

Source (you should point out proofs of your report, please give us some source): 
I think its easy reproducable, just go into Hyjal and Tank a Boss and pull him so several (melee)NPCs.

**Comment by Lyn3452** [comment]
Also this way you cant avoid triggering the cleave from Azgalor, since he should only does it when more than only the tank stand in front of the boss.

Proof: Cleave will not be cast if the Main Tank is the only target in front of Azgalor. With careful positioning of NPCs, and leaving pets to DPS the Lesser Doomguards, the amount of spike damage from Cleaves can be significantly reduced. Since the change to pets that keeps them behind a stationary target, if Azgalor is held still pets will likely not trigger cleaves.

from : https://wow.gamepedia.com/Azgalor_(tactics)

**Comment by pinkisnotgay** [comment]
@Afgann @Dootz @koostosh This needs urgent fix. Whenever you do Kaz'rogal all the mobs make boss do party-haste and just one shot the tank. )

**Comment by NeonicSupersonic** [comment]
Yup also Kaz'Rogal pbbly needs 10-15% hp nerf, we barely kill him with super heavy no-mele stacking. His stomp just too punishing for melees in general(and gl avoiding it with cripple slow when mt needs to be feed freedoms to keep threat up).
